source/opt/type_manager.h

```diff
@@ -120,6 +133,9 @@ class TypeManager {
  private:
```

Comment:
RegisterType comment needs clarification:

// If GetId(|type|) already returns a non-zero id, the return value will be  unchanged.


// If GetId(|type|) already returns a non-zero id, that mapping will not be changed.

Reply:
ok